engine/session_live.js

```diff
@@ -652,7 +644,6 @@ class SessionLive {
         // Wait a little before trying again
         debug(`[${this.sessionId}]: ALERT! Live Source Data NOT in sync! Will try again after ${retryDelayMs}ms`);
         await timer(retryDelayMs);
-        this.timerCompensation = false;
```

Comment:
This was originally introduced as an attempt to lower the chances of trying to fetch when the live source is "out-of-sync" with its media manifests. But a live source can still be unpredictable, so might not need it.  Removing it shouldn't be a problem.